### DISCUSSION TITLE
How to zoom into a window?

### DISCUSSION THREAD

**Discussion by mpas** [LazyVim/LazyVim#2356] [Q&A]
Is it possible to zoom into a window and later return? I happen to use DAP UI and the scope window is fairly small so was wondering how I could zoom into the scope window and later on return to the normal layout?

**Comment by dpetka2001** [comment]
You could try out [zen-mode](https://github.com/folke/zen-mode.nvim) to see if it meets your needs. There might be other plugins available that I'm not aware of.

**Reply by porobertdev** [reply]
For anyone that might still get here too:

Zoom **out**: `CTRL -`
Zoom-**in**: `CTRL SHIFT +`
